backend/productsearch/db/load.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+import asyncio
+import json
+import requests
+import os
+from typing import List
+
+import numpy as np
+from redisvl.index import AsyncSearchIndex
+
+from productsearch import config
+
+
+def read_from_s3():
+    res = requests.get(config.S3_DATA_URL)
+    return res.json()
+
+
+def read_product_json_vectors() -> List:
+    try:
+        with open(config.DATA_LOCATION + "/products.json") as f:
+            product_vectors = json.load(f)
+    except FileNotFoundError:
+        print("File not found, reading from S3")
+        product_vectors = read_from_s3()
```

Comment:
@tylerhutcherson I think I'd like to simplify this flow. Either load from local volume or pull from S3. If you pull from S3 I don't think we need to write that data because it's only needed to populate the db and causes bloat in the docker containers. 

I could add an additional script to pull this data into local for those developing locally but I think this is cleaner